src/parameters.ml

```diff
@@ -321,15 +318,15 @@ let[@inline] compute_current_q (last_q: fixedpoint) (last_drift: fixedpoint) (la
       target_{i+1} = FLOOR (q_{i+1} * index_{i+1} / kit_in_tez_{i+1})
     ]}
 *)
-let[@inline] compute_current_target (current_q: fixedpoint) (current_index: Ligo.tez) (current_kit_in_tez: ratio) : fixedpoint =
+let[@inline] compute_current_target (current_q: fixedpoint) (current_index: Ligo.nat) (current_kit_in_tez: ratio) : fixedpoint =
   let { num = num; den = den; } = current_kit_in_tez in
   fixedpoint_of_raw
     (fdiv_int_int
        (Ligo.mul_int_int
           den
           (Ligo.mul_int_int
              (fixedpoint_to_raw current_q)
-             (tez_to_mutez current_index)
+             (Ligo.int current_index)
```

Comment:
Sweet! Yeah, I hadn't thought of that—will do :+1: In fact, I believe there might be other places where this can apply; I'll make a separate search after merging this PR for such cases. That'll probably reduce the byte count too :ok_hand:

Reply:
Fixed this one in 89a5231, after we merge this PR I'll for more instances like this.